libmscore/staff.cpp

```diff
@@ -327,6 +327,19 @@ Fraction Staff::nextClefTick(const Fraction& tick) const
       return t != Fraction(-1,1) ? t : score()->endTick();
       }
 
+//---------------------------------------------------------
+//   Staff::currentClefTick
+//
+//    return the tick position of the clef currently
+//    in effect at tick
+//    return 0, if no such clef
+//---------------------------------------------------------
+
+Fraction Staff::currentClefTick(const Fraction& tick) const
```

Comment:
Does it make sense to pass fractions by reference? Fraction should be at most twice larger than a pointer (if both pointers and `int` have the same size, sometimes it isn't, like for g++ on my Debian system), and passing pointers make its dereferencing needed to operate on this fraction. Of course, references are not necessarily implemented as pointers but there doesn't seem to be much choice if compiler cannot inline the function.

Of course, the scope of this question is beyond this single PR.

Reply:
@dmitrio95 it seems to be done this way in most other places, so i just followed the general trend.

Reply:
Yes, this is a separate question, so I believe it is OK to leave it as is in this pull request. Still it would be interesting to see whether it makes any measurable performance difference, maybe I'll try to check it later.